src/main/java/frc/robot/modules/debug/ExtPrintStream.java

```diff
@@ -0,0 +1,105 @@
+package frc.robot.modules.debug;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+/** 
+ * En extension of PrintStream that splits the output between two streams
+*/
+public class ExtPrintStream extends PrintStream {
+    private final PrintStream second;
+    private final DateTimeFormatter dtf = DateTimeFormatter.ofPattern("[HH:mm:ss]");
+
+    public ExtPrintStream(OutputStream outputStream, PrintStream printStream)
+    {
+        super(outputStream);
+        this.second = printStream;
+    }
+
+    @Override
+    public void close() 
+    {
+        super.close();
+        second.close();
+    }
+
+    @Override
+    public void flush() 
+    {
+        super.flush();
+        second.flush();
+    }
+
+    @Override
+    public void write(byte[] buf, int off, int len) 
+    {
+        super.write(buf, off, len);
+        second.write(buf, off, len);
+    }
+
+    @Override
+    public void write(int b) 
+    {
+        super.write(b);
+        second.write(b);
+    }
+
+    @Override
+    public void write(byte[] b) throws IOException 
+    {
+        super.write(b);
+        second.write(b);
+    }
+
+    /** 
+    * Prints as normal to the second stream, then prints to the first stream with the applied time and info stamp
+    * @param x - String printed to both streams
+    * @throws IOException handled locally
+    */
```

Comment:
Possibly add an internal 'Level' state so that this can print other than than just the info level